src/OrbitGl/TrackManager.cpp

```diff
@@ -177,34 +176,52 @@ void TrackManager::SetFilter(const std::string& filter) {
   UpdateVisibleTrackList();
 }
 
+// This function assumes visible track list is updated.
+float TrackManager::GetVisibleTracksTotalHeight() const {
+  // Top and Bottom Margin. TODO: Margins should be treated in a different way (http://b/192070555).
+  float track_height = layout_->GetSchedulerTrackOffset() + layout_->GetBottomMargin();
```

Comment:
Nit: I find the name confusing because I thought this was a single track's height at first. Maybe rename it to `total_height`?